Sources/Mustache/Parser.swift

```diff
@@ -303,7 +303,7 @@ extension Parser {
 }
 
 /// context used in parser error
-public struct MustacheParserContext {
+public struct MustacheParserContext: Sendable {
```

Comment:
Why does this need to be Sendable

Reply:
MustacheParserContext is used as an inner property in `MustacheLibrary.ParserError` and `MustacheTemplate.ParserError`, and both of them conform to `Swift.Error`.
`Swift.Error` protocol conforms to `Sendable`. It is needed for synthesize conforming to `Sendable`.

```swift
extension MustacheTemplate {
    public struct ParserError: Swift.Error {
        public let context: MustacheParserContext
        public let error: Swift.Error
    }
...
}
```